src/components/modulesList/List.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
```

Comment:
Bardziej intuicyjne byłoby umieścić List.js w folderze view jako ModulesList, bo to już nie jest komponent tylko cały widok.

Reply:
Właśnie starałem się, żeby w samym widoku było jak najmniej rzeczy. Widok w moim przekonaniu ma dostarczyć dane do komponentów i powiedzieć im jak mają się zachowywać, bo komponenty powinny być głupie i nie mieć takich informacji, bo przez to mogą być reużywalne. Z tym, że ta Lista nie jest reużywalna, przez to, że ma taki guzik na dole. Nie wyobrażam sobie, że mogła być wykorzystana w inny sposób. Na razie zostawię to tak jak jest, bo będę przy tym grzebał tworząc kontekst i wtedy zobaczę co lepiej zrobić.